Resources/doc/events.rst

```diff
@@ -0,0 +1,81 @@
+see Doctrine's `Event Documentation`_.
+
+.. tip::
+
+    In addition to ODM events, you may also listen on lower-level MongoDB
+    events, which you'll find defined in the ``Doctrine\MongoDB\Events`` class.
+
+.. note::
+
+    Each connection in Doctrine has its own event manager, which is shared with
+    document managers tied to that connection. Listeners and subscribers may be
+    registered with all event managers or just one (using the connection name).
+
+In Symfony, you can register a listener or subscriber by creating a service
+and then `tagging`_ it with a specific tag.
+
+*   **event listener**: Use the ``doctrine_mongodb.odm.event_listener`` tag to
+    register a listener. The ``event`` attribute is required and should denote
+    the event on which to listen. By default, listeners will be registered with
+    event managers for all connections. To restrict a listener to a single
+    connection, specify its name in the tag's ``connection`` attribute.
+
+    The ``priority`` attribute, which defaults to ``0`` if omitted, may be used
+    to control the order that listeners are registered. Much like Symfony2's
+    `event dispatcher`_, greater numbers will result in the listener executing
+    first and listeners with the same priority will be executed in the order that
+    they were registered with the event manager.
+
+    Lastly, the ``lazy`` attribute, which defaults to ``false`` if omitted, may
+    be used to request that the listener be lazily loaded by the event manager
+    when its event is dispatched.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        services:
+            my_doctrine_listener:
+                class:   Acme\HelloBundle\Listener\MyDoctrineListener
+                # ...
+                tags:
+                    -  { name: doctrine_mongodb.odm.event_listener, event: postPersist }
+
+    .. code-block:: xml
+
+        <service id="my_doctrine_listener" class="Acme\HelloBundle\Listener\MyDoctrineListener">
+            <!-- ... -->
+            <tag name="doctrine_mongodb.odm.event_listener" event="postPersist" />
+        </service>
+
+    .. code-block:: php
+
+        $definition = new Definition('Acme\HelloBundle\Listener\MyDoctrineListener');
+        // ...
+        $definition->addTag('doctrine_mongodb.odm.event_listener', array(
+            'event' => 'postPersist',
+        ));
+        $container->setDefinition('my_doctrine_listener', $definition);
+
+*   **event subscriber**: Use the ``doctrine_mongodb.odm.event_subscriber`` tag
+    to register a subscriber. Subscribers are responsible for implementing
+    ``Doctrine\Common\EventSubscriber`` and a method for returning the events
+    they will observe. For this reason, this tag has no ``event`` attribute;
+    however, the ``connection``, ``priority`` and ``lazy`` attributes are
+    available.
+
+.. note::
+
+    Unlike Symfony2 event listeners, Doctrine's event manager expects each

```

Comment:
Change `Symfony2` to `Symfony`.

Reply:
Done. Also fixed a link to "tagging" part of docs